kube/resources/scheduler.yaml

```diff
@@ -69,17 +71,27 @@ spec:
                 configMapKeyRef:
                   name: airbyte-env
                   key: LOCAL_ROOT
-          resources:
-            limits:
-              memory: "256Mi"
-              cpu: "500m"
           ports:
             - containerPort: 8001
           volumeMounts:
             - name: airbyte-volume-configs
               mountPath: /configs
             - name: airbyte-volume-workspace
               mountPath: /workspace
+      initContainers:
```

Comment:
Is this going to erase existing configs if we restart the scheduler pod?

Reply:
You're right. Adding `-n` which will not clobber files that exist. Then if there are new integrations on an upgrade it will add them, but it won't overwrite anything that already exists.